apps/io-services-cms-webapp/src/utils/feature-flag-handler.ts

```diff
@@ -0,0 +1,51 @@
+// Handle Feature Flags
+import { ApiManagementClient } from "@azure/arm-apimanagement";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import * as TE from "fp-ts/lib/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+import { IConfig } from "../config";
+import {
+  getSubscription,
+  parseOwnerIdFullPath,
+} from "../lib/clients/apim-client";
+
+const wildcard = "*" as NonEmptyString;
+
+// Return True if the element is in the list or if the list contains the wildcard
+const isElementAllowedOnList =
+  (list: ReadonlyArray<NonEmptyString>) => (element: NonEmptyString) =>
+    list.includes(wildcard) || list.includes(element);
+
+/**
+ * Check if the owner of the service is enabled for CMS to Legacy Sync
+ * @param config
+ * @param apimClient
+ * @param serviceId
+ * @returns
+ */
+export const isUserEnabledForCmsToLegacySync = (
+  config: IConfig,
+  apimClient: ApiManagementClient,
+  serviceId: NonEmptyString
+) =>
```

Comment:
add return type to "enforce" type checking

Reply:
f5fb2fb